joomla/joomla-cms
# [users list modal] ismoo parameter need to still exist after form submit (solve #12122)

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/12122.
### Summary of Changes

The ismoo parameter is not in the form of users modal so it's lost after submit.
For more info see https://github.com/joomla/joomla-cms/issues/12122
### Testing Instructions
1. Code review
2. Apply patch on latest staging
3. Confirm there is no https://github.com/joomla/joomla-cms/issues/12122 issue 
### Documentation Changes Required

None.
